modules/op-specifying-pvc-as-volumesource-in-workspaces.adoc

```diff
@@ -0,0 +1,41 @@
+// This module is included in the following assembly:
+//
+//  *openshift_pipelines/creating-applications-with-cicd-pipelines.adoc
+
+[id="specifying-pvc-as-volumesource-in-workspaces_{context}"]
+= Specifying PersistentVolumeClaims as VolumeSource in Workspaces
+
+Workspaces help share data among Tasks and allow you to specify one or more volumes that each Task in the Pipeline requires during execution.
+
+In this section, you will create a PersistentVolumeClaim to provide data storage and bind it to the workspace. This PersistentVolumeClaim provides the volumes or filesystem required for the pipeline execution.
+
+[discrete]
+.Procedure
+
+. Copy the contents of the following sample PersistentVolumeClaim YAML file and save it:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: source-pvc
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 500Mi
+----
++
+. Create the PersistentVolumeClaim:
```

Comment:
For clarity:
s/. Create the PersistentVolumeClaim:/. Create the PersistentVolumeClaim, specifying the file you just created:/